### PR TITLE
Data driven boardroom jamie

### PR DESCRIPTION
**Pull request by JamieForster**
As requested during group coding session.

### CODE REVIEW

File: ios/Podfile [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Using a CDN with CocoaPods 1.7.2 or later can save a lot of time on pod installation, but it's experimental rather than the default.
+# source 'https://cdn.cocoapods.org/'
+
```
Comment (1solation) [inline ios/Podfile:4]: comment saying what this file is for at the top of this file (to run on actual IOS device etc)

File: ios/Podfile.lock [hunk 2]
```diff
@@ -0,0 +1,197 @@
+PODS:
```
Comment (1solation) [inline ios/Podfile.lock:2]: same as above review

File: lib/pages/home.dart [hunk 3]
```diff
@@ -1,72 +1,119 @@
...
+                leading: Icon(Icons.local_hospital, size: 50, color: Colors.white),
+                title: Text(snapshot["boardName"], style: new TextStyle(color: Colors.white)),
+                subtitle: Text(snapshot.documentID, style: new TextStyle(color: Colors.white70)),
+                onTap: (){
+                  Navigator.push(
+                      context,
+                      MaterialPageRoute(
+                        builder: (context) => Boardroom(user: user, document: snapshot),
+                      )
+                  ); // Navigator.push
```
Comment (1solation) [inline lib/pages/home.dart]: remove comment

File: lib/pages/home.dart [hunk 4]
```diff
@@ -1,72 +1,119 @@
...
+  //TODO: Redo theme related code to be dynamic, not hard-coded.
+
+  @override
+  Widget build(BuildContext context){
+    return _createList(context);
+  }
+
+  Scaffold _createList(BuildContext context){
+
+    print('Stream Builder');
```
Comment (1solation) [inline lib/pages/home.dart]: type debug into print statement

File: lib/pages/home.dart [hunk 5]
```diff
@@ -1,72 +1,119 @@
...
+    ),
+      body: Container(
+        padding: EdgeInsets.all(20.0),
+        child: StreamBuilder(
+          stream: Firestore.instance.collection('boardroom').document(widget.document.documentID).collection('polls').snapshots(),
+          builder: (context, AsyncSnapshot snapshot) {
+            if(!snapshot.hasData){
+              return Text("Loading...");
+            } else {
+              print("else Has Data!");
```
Comment (1solation) [inline lib/pages/home.dart]: remove print

File: lib/pages/home.dart [hunk 6]
```diff
@@ -1,72 +1,119 @@
...
+              print(snapshot.data.documents.length);
```
Comment (1solation) [inline lib/pages/home.dart]: add debug print statement in here OR before

**Review by 1solation — CHANGES_REQUESTED** [review]
beautiful work boyz, we da team

File: lib/pages/home.dart (hunk 6 above)
Comment (JamieForster) [reply lib/pages/home.dart]: 👍

File: lib/pages/home.dart (hunk 5 above)
Comment (JamieForster) [reply lib/pages/home.dart]: 👍

File: lib/pages/home.dart (hunk 4 above)
Comment (JamieForster) [reply lib/pages/home.dart]: 👍

File: lib/pages/home.dart (hunk 3 above)
Comment (JamieForster) [reply lib/pages/home.dart]: 👍

File: ios/Podfile (hunk 1 above)
Comment (JamieForster) [reply ios/Podfile:4]: 👍

File: ios/Podfile.lock (hunk 2 above)
Comment (JamieForster) [reply ios/Podfile.lock:2]: 👍

**Review by 1solation — APPROVED** [review]
looks good

**Pull request merged**